examples/cpp/minimal/CMakeLists.txt

```diff
@@ -4,24 +4,25 @@ cmake_minimum_required(VERSION 3.16)
 # where the rerun_c build is to be found by setting the `RERUN_CPP_URL` variable.
 # This can be done by passing `-DRERUN_CPP_URL=<path to rerun_sdk_cpp zip>` to cmake.
 if(DEFINED RERUN_REPOSITORY)
-    add_executable(example_minimal main.cpp)
-    set_default_warning_settings(example_minimal)
+    add_executable(example main.cpp)
```

Comment:
there's an "examples" target that encompasses all cpp examples making this a bit _meh_ within the rerun repo itself. Maybe we can land at `example_rerun`? Still though when you list all targets `example_minimal` is kinda nice

Reply:
Ahh fair, I forgot about this in the context of the outer project. In that case I'm going to use `example_minimal` in the tutorial.